pkgs/development/libraries/libvoikko/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, autoreconfHook
+, fetchFromGitHub
+, pkg-config
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libvoikko";
+  version = "4.3.2";
+
+  src = fetchFromGitHub {
+    owner = "voikko";
+    repo = "corevoikko";
+    rev = "refs/tags/rel-libvoikko-${version}";
+    hash = "sha256-0MIQ54dCxyAfdgYWmmTVF+Yfa15K2sjJyP1JNxwHP2M=";
+  };
+
+  configureFlags = [ "--disable-hfst" ];
```

Comment:
Why do we disable this if the library is available?

Reply:
Looks like it was just a leftover before I packaged hfst-ospell, I removed it now